cmd/ipfs_routingd/.go-ipfs/config

```diff
@@ -0,0 +1,60 @@
+  "Identity": {
+    "PeerID": "QmbuseHCCV8emQgTmB7Kifzm5MMsWP7seW6rwoJQE8LEic",
+    "PrivKey": "CAAS4AQwggJcAgEAAoGBALfEh2ISGmscQ0pOHgVTR6jCvVsLghlizCxmbThdLZMeHfVkzIk4IOtqEbXGb8fENAgC7DHSZTqHplJ884pxEE0NGOICpQe3g2KW/ltuRlLYvjOhBTn6ks5HKMjwPtibQNGiRtn8bsJfYHj7mx4dcxXuSQY7gjnorEmj7RMdsLaJAgMBAAECgYA1cSb6A0/BhIehLVSXsd9uGawdtRwRq+S6qNLsiFR/q0uFGNpOsY8JR2B9H4dUbh+/kLIFzyz3N4nMBWCCC110vJXS3whUeH7fvAoBBx2uYR13F63IHHDTub6TEhG/3qPO/E29SvqOVzYhuD3s1U4hnOuR8BV5fsVU4qfd7D33AQJBAOWJdLw2KxNKwkih061F4kERVatgbDmtcQsRxkueJyI9GPGJ4TvwXBvzwW4F87uMQuOwMCB8jtSKGXCebVhDIWkCQQDM9D26VblRV15v1mJfAYVqLumhR3svJXx4RkepTfCOtiS1dNxfu1l3QJgJYGPsAntexdJqmd5wcL469A32KSghAkBxWSaPRsSQu0sjqN4p46et8Ax5G512AgMfr0BX2RqEGt21cRwLiE9D8mQjm8bXZ5UkcL3jXP7/T87uQR3IByZ5AkAUHJd+FHFl8PvcX54xknWkEMxuPhRxcO1A5CyeCX7kki0R/VCuuDS0RgtlSCcl/qxahXD7cvPn9rhEErTYBfvhAkEAhoFcr5s8ge/6qwP2hvcZfKPubk4G7xdH0xDs5Tff1sbkcYoYXyU1cvZJnyvI0XSyJV6vsdxFYqI6kwD/TE5H4Q=="
+  },
+  "Datastore": {
+    "Type": "leveldb",
+    "Path": "/Users/btc/go/src/github.com/jbenet/go-ipfs/cmd/ipfs_routingd/.go-ipfs/datastore"
+  },
+  "Addresses": {
+    "Swarm": [
+      "/ip4/0.0.0.0/tcp/4002"
+    ],
+    "API": "/ip4/127.0.0.1/tcp/5001",
+    "Gateway": "/ip4/127.0.0.1/tcp/8080"
+  },
+  "Mounts": {
+    "IPFS": "/ipfs",
+    "IPNS": "/ipns"
+  },
+  "Version": {
+    "Current": "0.1.7",
+    "Check": "error",
+    "CheckDate": "0001-01-01T00:00:00Z",
+    "CheckPeriod": "172800000000000",
+    "AutoUpdate": "minor"
+  },
+  "Bootstrap": [
+    "/ip4/104.131.131.82/tcp/4001/ipfs/QmaCpDMGvV2BGHeYERUEnRQAwe3N8SzbUtfsmvsqQLuvuJ",
+    "/ip4/104.236.176.52/tcp/4001/ipfs/QmSoLnSGccFuZQJzRadHn95W2CrSFmZuTdDWP8HXaHca9z",
+    "/ip4/104.236.179.241/tcp/4001/ipfs/QmSoLpPVmHKQ4XTPdz8tjDFgdeRFkpV8JgYq8JVJ69RrZm",
+    "/ip4/162.243.248.213/tcp/4001/ipfs/QmSoLueR4xBeUbY9WZ9xGUUxunbKWcrNFTDAadQJmocnWm",
+    "/ip4/128.199.219.111/tcp/4001/ipfs/QmSoLSafTMBsPKadTEgaXctDQVcqN88CNLHXMkTNwMKPnu",
+    "/ip4/104.236.76.40/tcp/4001/ipfs/QmSoLV4Bbm51jM9C4gDYZQ9Cy3U6aXMJDAbzgu2fzaDs64",
+    "/ip4/178.62.158.247/tcp/4001/ipfs/QmSoLer265NRgSp2LA3dPaeykiS1J6DifTC88f5uVQKNAd",
+    "/ip4/178.62.61.185/tcp/4001/ipfs/QmSoLMeWqB7YGVLJN3pNLQpmmEk35v6wYtsMGLzSr5QBU3",
+    "/ip4/104.236.151.122/tcp/4001/ipfs/QmSoLju6m7xTh3DuokvT3886QRYqxAzb1kShaanJgW36yx"
+  ],
+  "Tour": {
+    "Last": ""
+  },
+  "Gateway": {
+    "RootRedirect": "",
+    "Writable": false
+  },
+  "SupernodeRouting": {
+    "Servers": [
+      "/ip4/104.236.176.52/tcp/4002/ipfs/QmXdb7tWTxdFEQEFgWBqkuYSrZd3mXrC7HxkD4krGNYx2U",
+      "/ip4/104.236.179.241/tcp/4002/ipfs/QmVRqViDByUxjUMoPnjurjKvZhaEMFDtK35FJXHAM4Lkj6",
+      "/ip4/104.236.151.122/tcp/4002/ipfs/QmSZwGx8Tn8tmcM4PtDJaMeUQNRhNFdBLVGPzRiNaRJtFH",
+      "/ip4/162.243.248.213/tcp/4002/ipfs/QmbHVEEepCi7rn7VL7Exxpd2Ci9NNB6ifvqwhsrbRMgQFP",
+      "/ip4/128.199.219.111/tcp/4002/ipfs/Qmb3brdCYmKG1ycwqCbo6LUwWxTuo3FisnJV2yir7oN92R"
+    ]
+  },
+  "Log": {
+    "MaxSizeMB": 250,
+    "MaxBackups": 1,
+    "MaxAgeDays": 0
+  }
```

Comment:
if the config file is here just for the routing server addrs, maybe those can be added in a sh script, or hardcoded into the snr code.